### DISCUSSION TITLE
An Ask Box Feature I Miss

### DISCUSSION THREAD

**Discussion by DuderComputer** [AprilSylph/XKit-Rewritten#333] [Suggestion Box]
XKit had an enhanced inbox plugin that was great, but obviously was mostly depreciated by Tumblr's new post editor, which finally enables formatting options and queuing that you used to need XKit to get.

But one thing the new post editor doesn't support is automatically tagging inbox posts. So, I run an ask blog that doesn't always post ask box content, so it was useful to use the XKit plugin to automatically tag the ask box post with #questions and a tag for the asker's username.

With the death of old New XKit and the new post editor breaking the plugin anyway, I've been having to type those tags out manually for every post I queue. Not the end of the universe, but ever so slightly more inconvenient. 

It'd be nice if that could come back somehow.

**Comment by AprilSylph** [comment]
This is planned for Quick Tags. In the meantime, you should be able to create a tag bundle for your static answer tags.

I believe automatically tagging with the asker's username should be possible, but I won't know until I try.

**Reply by AprilSylph** [reply]
Quick Tags will be receiving an option for an automatic custom answer tag and an option to tag the asker's username automatically when answering in v0.11.0, which I hope to release tomorrow.

**Reply by DuderComputer** [reply]
That's great to hear! Thank you so much.
